src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentOrderApprovedDocumentService.java

```diff
@@ -37,6 +54,63 @@ public CaseDocument generateApprovedConsentOrderLetter(CaseDetails caseDetails,
                 config.getApprovedConsentOrderFileName());
     }
 
+    public CaseDocument generateApprovedConsentOrderNotificationLetter(CaseDetails caseDetails, String authToken) {
+        log.info("Generating Approved Consent Order Notification Letter {} from {} for bulk print",
+                config.getApprovedConsentOrderFileName(),
+                config.getApprovedConsentOrderTemplate());
+
+        prepareApprovedConsentOrderNotificationLetter(caseDetails);
+
+        return generateDocument(authToken, caseDetails,
+                config.getApprovedConsentOrderNotificationTemplate(),
+                config.getApprovedConsentOrderNotificationFileName());
+    }
+
+    private void prepareApprovedConsentOrderNotificationLetter(CaseDetails caseDetails) {
+        Map<String, Object> caseData = caseDetails.getData();
+        Map addressToSendTo;
+
+        String ccdNumber = nullToEmpty((caseDetails.getId()));
+        String reference = "";
+        String addresseeName;
+        String applicantName = nullToEmpty((caseData.get(APPLICANT_FIRST_MIDDLE_NAME)))
+                + " " + nullToEmpty((caseData.get(APPLICANT_LAST_NAME)));
+        String respondentName = nullToEmpty((caseData.get(APP_RESPONDENT_FIRST_MIDDLE_NAME)))
+                + " " + nullToEmpty((caseData.get(APP_RESPONDENT_LAST_NAME)));
+        String applicantRepresented = nullToEmpty(caseData.get(APPLICANT_REPRESENTED));
+
+        if (applicantRepresented.equalsIgnoreCase(YES_VALUE)) {
```

Comment:
I think you should flip this statement (to avoid null pointer exceptions).

Reply:
I'd absolutely love to have the field respSolicitorAddress changed but unfortunately it's already being used in CCD. I've just created the constant, not the initial implementation